core/src/main/java/com/salesforce/slds/validation/validators/impl/recommendation/MobileFriendlyValidator.java

```diff
@@ -34,6 +34,11 @@
 @Component
 @Lazy
 public class MobileFriendlyValidator implements RecommendationValidator, InitializingBean {
+    public static final String NON_MOBILE_FRIENDLY_MESSAGE_TEMPLATE =
```

Comment:
If you think this message may change in the future, you may want to consider using [MessageFormat](https://docs.oracle.com/javase/7/docs/api/java/text/MessageFormat.html). Otherwise, this is good.

Reply:
I'm not sure if/when this message might change in the future.